frontend/src/index.scss

```diff
@@ -1,267 +1,94 @@
-$color-bg: #fff;
+@import 'autocomplete-theme-classic';
+
 $color-muted: #969faf;
 $color-light: #797979;
-$color-text: #23263b;
 $color-mark: rgb(84, 104, 255);
-$color-bg-selected: #f5f5fa;
-$color-input-icon: #777;
-$color-source-icon: rgba(80, 80, 80, 0.32);
 
 $font-size-xs: 12px;
 $font-size-s: 14px;
 $font-size-m: 16px;
 
+$size-xs: 2px;
 $size-s: 4px;
 $size-m: 8px;
 $size-l: 16px;
 $size-xl: 32px;
 
-$height: $size-xl;
-$height-icon: $size-l;
 $font-size: $font-size-m;
 
-.aa-Autocomplete, .aa-Panel {
+// ---- Overridden definitions of classic theme
+:root {
   --color-mark: #{$color-mark};
-  --color-background: #{$color-bg};
-  --color-selected: #{$color-bg-selected};
-  --color-text: #{$color-text};
-  --color-input-icon: #{$color-input-icon};
-  --color-source-icon: #{$color-source-icon};
-  --height: #{$height};
-  --height-icon: #{$height-icon};
-  --font-size: #{$font-size};
```

Comment:
CSS variable are part of the exposed API, I would prefer to keep them (even if it's the same output than `@import 'autocomplete-theme-classic';`